apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimFissRead.json

```diff
@@ -70,9 +71,9 @@
   ],
```

Comment:
Could we add a non-null meda-prov-6 value to this fiss claim so we can test its output in the claimFissRead.json